modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/ViewObjectDefinitions.tsx

```diff
@@ -325,32 +321,19 @@ export default function ViewObjectDefinitions({
 	};
 
 	useEffect(() => {
-		if (Liferay.FeatureFlags['LPS-148856']) {
-			const makeFetch = async () => {
-				API.getAllObjectFolders().then((response) => {
-					setObjectFoldersRequestInfo(response);
-					setSelectedObjectFolder(response.items[0]);
-					setLoading(false);
-				});
-
-				const objectDefinitions = await API.getAllObjectDefinitions();
-
-				setObjectDefinitionActions(objectDefinitions.actions);
-			};
+		const makeFetch = async () => {
+			API.getAllObjectFolders().then((response) => {
+				setObjectFoldersRequestInfo(response);
+				setSelectedObjectFolder(response.items[0]);
+				setLoading(false);
+			});
 
-			makeFetch();
-		}
-
-		Liferay.on('addObjectDefinition', () =>
-			setShowModal((previousState: ViewObjectDefinitionsModals) => ({
-				...previousState,
-				addObjectDefinition: true,
-			}))
-		);
```

Comment:
Hey @p-albuquerque, this section is responsible for opening the modal to add a new object definition.